script.js

```diff
@@ -122,7 +122,10 @@ const getStatus = id => _.find(statusOptions, s => (s.id === id.toLowerCase()))
 const createListItem = (location, status, lng, lat) => {
   const urgentNeed = location.urgentNeed ? `<p class="urgentNeed p location-list--important">Urgent Need: ${location.urgentNeed}</p>` : ''
   const seekingMoney = location.seekingMoney ? `<span class="seekingMoney location-list--badge">Needs Money Donations</span>` : ''
-  const seekingVolunteers = location.seekingVolunteers ? `<span class="seekingVolunteers location-list--badge">Needs Volunteer Support</span>` : ''
+  let seekingVolunteers = ''
+  if (location.seekingVolunteers && location.seekingVolunteers.match(/(?:\byes\b)/i)) {
```

Comment:
I would've been lazy and tried to just detect "yes" in a string but you were adamant that "yesterday" not trigger the button.